engine-rest/engine-rest-openapi/src/main/templates/lib/commons/task-query-params.ftl

```diff
@@ -657,5 +657,14 @@
   <@lib.parameter name = "parentTaskId"
       location = "query"
       type = "string"
+      desc = "Restrict query to all tasks that are sub tasks of the given task. Takes a task id." />
+
+  <@lib.parameter name = "withCommentAttachmentInfo"
```

Comment:
Note to myself: the parameter will be confusing for "count" queries. Do we need to document this?

Reply:
Could you please add this to the description ⬆️ the param is not considered for count queries.